### PR TITLE
Update sample repo and lint

### PR DESCRIPTION
**Pull request by ilikesymmetry**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: docs/pages/guides/spend-permissions/overview.mdx [hunk 1]
```diff
@@ -28,33 +29,29 @@ struct SpendPermission {
...
 
 ## Approving
 
 A user approves a spend permission by signing an [ERC-712](https://eips.ethereum.org/EIPS/eip-712) typed object that contains
-the spend permission properties. This signature and the corresponding spend permission details are then submitted to 
+the spend permission properties. This signature and the corresponding spend permission details are then submitted to
 `SpendPermissionManager.approveWithSignature` to approve the spend permission onchain.
 
-Multiple spend permissions can be batched and approved with a single signature by constructing a batch spend permission object
-and approving with `SpendPermissionManager.approveBatchWithSignature`.
```
Comment (ilikesymmetry) [inline docs/pages/guides/spend-permissions/overview.mdx:41]: intentional removal to not bring up batching

File: docs/pages/guides/spend-permissions/quick-start.mdx [hunk 2]
```diff
@@ -2,22 +2,26 @@ import { Callout } from "vocs/components";
 
 # Getting Started with Spend Permissions
 
-This guide will walk you through a simple example of building an app that leverages Spend Permissions using 
+This guide will walk you through a simple example of building an app that leverages Spend Permissions using
 [OnchainKit](https://onchainkit.xyz/), [Viem](https://viem.sh/) and [Wagmi](https://wagmi.sh/).
 
-A complete example of this demo can be found [here](https://github.com/ilikesymmetry/subscribe-onchain).  
+A complete example of this demo can be found [here](https://github.com/ilikesymmetry/spend-permissions-quickstart).
```
Comment (ilikesymmetry) [inline docs/pages/guides/spend-permissions/quick-start.mdx:8]: new repo name to be more clear that it is meant to be a complete version of this quickstart guide

**Comment by ilikesymmetry** [comment]
something got messed up with rebase, restarting

**Pull request closed without merging**